datasets/index.md

```diff
@@ -3,7 +3,6 @@ layout: default
 title: Data Sources
 nav_order: 110
 has_children: true
-has_toc: false
```

Comment:
is this an intentional change?

Reply:
Oh oops, no, it is what was generated by the script.  Should I revert this?

Reply:
hm, looks like there is no difference in output. it would make sense to be false by default so this is ok to remove